### PR TITLE
Dennisc nc news 2

### PR DESCRIPTION
**Pull request by DennisTockan**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+describe("GET /api/topics", () => {
+  test("200 responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics") // path
+      .expect(200) // successful request
+      .then(({ body }) => {
+        // this deconstructs the data to get the key of body
+        const { topics } = body;
+        expect(topics).toHaveLength(3);
```
Comment (johnhutcheon) [inline __tests__/app.test.js:28]: nice work asserting the length before a forEach! 👍

**Comment by DennisTockan** [comment]
is this ready to be merged or what steps do i do?

File: app.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+} = require("./errors");
+
+app.get("/api/topics", getAllTopics);
+
+app.use(handlePsqlErrors);
+
+app.use(handleCustomErrors);
+
+app.use(handleServersErrors);
+
```
Comment (johnhutcheon) [inline app.js:17]: Really great work here! 😄

**Review by johnhutcheon — APPROVED** [review]
Hi Dennis! This all looks really good, happy for you to merge! 👍

**Pull request merged**